CommonTools/PileupAlgos/plugins/PuppiPhoton.cc

```diff
@@ -21,7 +21,7 @@
 #include "DataFormats/Common/interface/Association.h"
 #include "DataFormats/Math/interface/deltaR.h"
 //Main File
-#include "CommonTools/PileupAlgos/plugins/PuppiPhoton.h"
+#include "CommonTools/PileupAlgos/interface/PuppiPhoton.h"
```

Comment:
Why moving the headers to ```/interface```?
Please, bring them back to ```\plugin```

Reply:
done